recipes/parallelproj/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.5" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 411fe7552106db2e20487f2429907bfac641020469b0e7593134975345148cf5
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  skip: True  # [win]
+  skip: True  # [osx]
```

Comment:
```suggestion
  skip: True  # [osx or win]
  # only works for CUDA>=11.0
  skip: True  # [linux and cuda_compiler_version == "10.2"]
```

Reply:
Regarding your [question](https://gitter.im/conda-forge/conda-forge.github.io?at=637558744fee72663c7b6bac)

Reply:
Thanks for the code review :). Changes were included